src/components/Feed.jsx

```diff
@@ -0,0 +1,67 @@
+import moment from "moment"
+import PropTypes from "prop-types"
+import { useState } from "react"
+import "./Feed.css"
+
+export const Feed = ({ thoughts, fetchHappyThoughts }) => {
+  const [likes, setLikes] = useState({})
+  const [likedThoughts, setLikedThoughts] = useState([])
+
+  const onLikeIncrease = (_id) => {
+    // Send a request to the API to increase the hearts for the post with that ID
+    fetch(
+      `https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts/${_id}/like`,
+      { method: "POST" }
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        //Update the local state to reflect the new like count
+        setLikes((prevLikes) => ({
+          ...prevLikes,
+          [_id]: data.hearts,
+        }))
+        setLikedThoughts((prevLikedThoughts) => [...prevLikedThoughts, _id])
+        fetchHappyThoughts()
+      })
+      .catch((error) => {
+        console.error("Error increasing like:", error)
+      })
+  }
+
+  const isThoughtLiked = (_id) => {
+    return likedThoughts.includes(_id)
```

Comment:
Since the likes state is an object that includes the id and hearts of liked posts, it seems unnecessary to have likedThoughts state created. You can use "_id in likes" to check if the likes have a particular id as a key.